docker/keycloak/README.md

```diff
@@ -19,7 +19,7 @@ is intended for a SMART on FHIR app with patient scopes.
 - The `alvearie/keycloak-config:latest` docker image to configure a SMART
 enabled realm. This is useful for the `patient` access-checker.
 
-- The `gcr.io/second-scion-309318/keycloak-config:latest` docker image to configure a realm
+- The `us-docker.pkg.dev/fhir-proxy-build/stable/keycloak-config:latest` docker image to configure a realm
```

Comment:
nit: It would be nice to keep line lengths short (e.g., below 80 or 100); we should add automatic style checker for `.md` files too :-)

Reply:
I will fix this in a separate PR. Already have a solution. Just going to merge this first